### PR TITLE
[Docs] Add Chinese version of customize_losses

### PR DESCRIPTION
**Pull request by wangbo-zhao**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

This PR is for chinese version of tutorials/customize_losses.md

### CODE REVIEW

File: docs_zh-CN/tutorials/customize_losses.md [hunk 1]
```diff
@@ -1 +1,92 @@
 # 教程 6: 自定义损失函数
+
+MMDetection为用户提供了不同的损失函数。但是默认的配置可能无法适应于不同的数据集和模型，所以用户可能会希望修改某一个损失函数来适应新的情况。
```
Comment (BIGWangYuDong) [inline docs_zh-CN/tutorials/customize_losses.md:3]: space
中文 English 中文。 Need space between Chinese and English

File: docs_zh-CN/tutorials/customize_losses.md [hunk 2]
```diff
@@ -1 +1,92 @@
...
+
+本教程首先详细的解释计算损失的过程然后给出一些关于如何修改每一个步骤的指导。对损失的修改可以被分为微调和加权。
+
+
+## 一个损失的计算过程
+给定输入预测和目标，以及权重，损失函数会把输入的张量映射到最后的损失标量。映射过程可以分为下面四个步骤：
```
Comment (BIGWangYuDong) [inline docs_zh-CN/tutorials/customize_losses.md:9]: 给定输入预测和目标 不通顺，给定的输入，包括预测和目标，可能会好一点，如果有更好的可以替换

File: docs_zh-CN/tutorials/customize_losses.md [hunk 3]
```diff
@@ -1 +1,92 @@
...
+1. 通过损失核函数获取**元素**或者**样本**损失。
```
Comment (BIGWangYuDong) [inline docs_zh-CN/tutorials/customize_losses.md:10]: also can add bold the corresponding text in English docs.

File: docs_zh-CN/tutorials/customize_losses.md [hunk 4]
```diff
@@ -1 +1,92 @@
...
+2. 通过权重张量来给损失**逐像素**权重。
```
Comment (BIGWangYuDong) [inline docs_zh-CN/tutorials/customize_losses.md:11]: 需要统一 `逐元素`

File: docs_zh-CN/tutorials/customize_losses.md [hunk 5]
```diff
@@ -1 +1,92 @@
...
+loss_cls=dict(
+    type='FocalLoss',
+    use_sigmoid=True,
+    gamma=2.0,
+    alpha=0.25,
+    loss_weight=1.0)
+```
+
+### 微调超参数（步骤1）
+`gamma`和`beta`是Focal Loss中的两个超参数。如果我们想把`gamma`的值设为1.5，把`alpha`的值设为0.5，我们可以在配置文件中按照如下指定：
```
Comment (BIGWangYuDong) [inline docs_zh-CN/tutorials/customize_losses.md:41]: 中文 数字 中文 
need check this PR

File: docs_zh-CN/tutorials/customize_losses.md [hunk 6]
```diff
@@ -1 +1,92 @@
 # 教程 6: 自定义损失函数
+
+MMDetection为用户提供了不同的损失函数。但是默认的配置可能无法适应于不同的数据集和模型，所以用户可能会希望修改某一个损失函数来适应新的情况。
+
+本教程首先详细的解释计算损失的过程然后给出一些关于如何修改每一个步骤的指导。对损失的修改可以被分为微调和加权。
+
+
+## 一个损失的计算过程
```
Comment (BIGWangYuDong) [inline docs_zh-CN/tutorials/customize_losses.md:8]: 增加空行，标题和文本之间要有空行

File: docs_zh-CN/tutorials/customize_losses.md (hunk 1 above)
Comment (BIGWangYuDong) [reply docs_zh-CN/tutorials/customize_losses.md:3]: 1. MMDetection 为用户。。。
2. 但是默认的配置可能无法适应不同的数据

File: docs_zh-CN/tutorials/customize_losses.md [hunk 7]
```diff
@@ -1 +1,107 @@
...
+2. 通过权重张量来给损失**逐元素**权重。
+
+3. 把损失张量归纳为一个**标量**。
+
+4. 用一个**张量**给当前损失一个权重。
+
+
+## 微调损失
+
+微调一个损失主要与步骤1，3，4有关，大部分的修改可以在配置文件中指定。这里我们用[Focal Loss (FL)](https://github.com/open-mmlab/mmdetection/blob/master/mmdet/models/losses/focal_loss.py)作为例子。
```
Comment (BIGWangYuDong) [inline docs_zh-CN/tutorials/customize_losses.md:23]: 主要与步骤 1，3，4 有关

File: docs_zh-CN/tutorials/customize_losses.md [hunk 8]
```diff
@@ -1 +1,107 @@
...
+下面的代码分别是构建FL的方法和它的配置文件，他们是一一对应的。
```
Comment (BIGWangYuDong) [inline docs_zh-CN/tutorials/customize_losses.md:24]: 构建 FL 的方法

File: docs_zh-CN/tutorials/customize_losses.md [hunk 9]
```diff
@@ -1 +1,107 @@
...
+    type='FocalLoss',
+    use_sigmoid=True,
+    gamma=2.0,
+    alpha=0.25,
+    loss_weight=1.0)
+```
+
+### 微调超参数（步骤1）
+
+`gamma`和`beta`是Focal Loss中的两个超参数。如果我们想把`gamma`的值设为 1.5，把`alpha`的值设为 0.5，我们可以在配置文件中按照如下指定：
```
Comment (BIGWangYuDong) [inline docs_zh-CN/tutorials/customize_losses.md:49]: `gamma` 和 `beta` 是 Focal Loss 中的两个超参数.。如果我们想把 `gamma` 的值设为 1.5，把 `alpha` 的值设为 0.5，

File: docs_zh-CN/tutorials/customize_losses.md [hunk 10]
```diff
@@ -1 +1,107 @@
...
+    type='FocalLoss',
+    use_sigmoid=True,
+    gamma=1.5,
+    alpha=0.5,
+    loss_weight=1.0)
+```
+
+### 微调归纳方式（步骤3）
+
+Focal Loss默认的归纳方式是`mean`。如果我们想把归纳方式从`mean`改成`sum`，我们可以在配置文件中按照如下指定：
```
Comment (BIGWangYuDong) [inline docs_zh-CN/tutorials/customize_losses.md:62]: Focal Loss 默认的归纳方式是 `mean`。如果我们想把归纳方式从 `mean` 改成 `sum`，我们可以在配置文件中按照如下指定：

File: docs_zh-CN/tutorials/customize_losses.md [hunk 11]
```diff
@@ -1 +1,107 @@
...
+    use_sigmoid=True,
+    gamma=2.0,
+    alpha=0.25,
+    loss_weight=0.5)
+```
+
+## 加权损失（步骤2）
+
+加权损失就是我们逐元素修改损失权重。更具体来说，我们给损失张量乘以一个与他有相同形状的权重张量。所以，损失中不同的元素可以被赋予不同的比例，所以这里叫做逐元素。
+损失的权重在不同模型中变化很大，而且与上下文相关，但是总的来说主要有两种损失权重：分类损失的`label_weights`和边界框的`bbox_weights`。你可以在相应的头中的`get_target`方法中找到他们。这里我们使用[ATSSHead](https://github.com/open-mmlab/mmdetection/blob/master/mmdet/models/dense_heads/atss_head.py#L530) 作为一个例子。它继承了[AnchorHead](https://github.com/open-mmlab/mmdetection/blob/master/mmdet/models/dense_heads/anchor_head.py)，但是我们重写它的
```
Comment (BIGWangYuDong) [inline docs_zh-CN/tutorials/customize_losses.md:90]: space

File: docs_zh-CN/tutorials/customize_losses.md [hunk 12]
```diff
@@ -1 +1,107 @@
...
+`get_targets` 方法来产生不同的`label_weights` 和 `bbox_weights`。
```
Comment (BIGWangYuDong) [inline docs_zh-CN/tutorials/customize_losses.md:91]: 合并到上一段，注意空格

**Review by BIGWangYuDong — COMMENTED** [review]
Please fix the review one by one and click `resolve`

File: docs_zh-CN/tutorials/customize_losses.md [hunk 13]
```diff
@@ -1 +1,106 @@
 # 教程 6: 自定义损失函数
+
+MMDetection 为用户提供了不同的损失函数。但是默认的配置可能无法适应不同的数据和模型，所以用户可能会希望修改某一个损失函数来适应新的情况。
+
+本教程首先详细的解释计算损失的过程然后给出一些关于如何修改每一个步骤的指导。对损失的修改可以被分为微调和加权。
+
+
```
Comment (BIGWangYuDong) [inline docs_zh-CN/tutorials/customize_losses.md:7]: delete this line

**Review by BIGWangYuDong — COMMENTED** [review]
LGTM

**Review by BIGWangYuDong — COMMENTED** [review]
I'm sorry that I make you confused. What I mean is you need to delete the Blank line, not the text.  ┑(￣Д ￣)┍

File: docs_zh-CN/tutorials/customize_losses.md (hunk 13 above)
Comment (BIGWangYuDong) [reply docs_zh-CN/tutorials/customize_losses.md:5]: I'm sorry that I make you confused. What I mean is you need to delete the Blank line, not the text

File: docs_zh-CN/tutorials/customize_losses.md [hunk 14]
```diff
@@ -1 +1,104 @@
...
+    type='FocalLoss',
+    use_sigmoid=True,
+    gamma=1.5,
+    alpha=0.5,
+    loss_weight=1.0)
+```
+
+### 微调归纳方式（步骤3）
+
+Focal Loss默认的归纳方式是 `mean`。如果我们想把归纳方式从 `mean` 改成 `sum`，我们可以在配置文件中按照如下指定：
```
Comment (AronLin) [inline docs_zh-CN/tutorials/customize_losses.md:60]: Space after `Focal Loss`

File: docs_zh-CN/tutorials/customize_losses.md [hunk 15]
```diff
@@ -1 +1,104 @@
...
+2. 通过权重张量来给损失**逐元素**权重。
+
+3. 把损失张量归纳为一个**标量**。
+
+4. 用一个**张量**给当前损失一个权重。
+
+
+## 微调损失
+
+微调一个损失主要与步骤1，3，4有关，大部分的修改可以在配置文件中指定。这里我们用 [Focal Loss (FL)](https://github.com/open-mmlab/mmdetection/blob/master/mmdet/models/losses/focal_loss.py) 作为例子。
```
Comment (AronLin) [inline docs_zh-CN/tutorials/customize_losses.md:21]: space before and after numbers

File: docs_zh-CN/tutorials/customize_losses.md [hunk 16]
```diff
@@ -1 +1,105 @@
 # 教程 6: 自定义损失函数
+
+MMDetection 为用户提供了不同的损失函数。但是默认的配置可能无法适应不同的数据和模型，所以用户可能会希望修改某一个损失函数来适应新的情况。
+本教程首先详细的解释计算损失的过程然后给出一些关于如何修改每一个步骤的指导。对损失的修改可以被分为微调和加权。
```
Comment (BIGWangYuDong) [inline docs_zh-CN/tutorials/customize_losses.md:4]: *空行*
本教程首先详细的解释计算损失的过程然后给出一些关于如何修改每一个步骤的指导。对损失的修改可以被分为微调和加权。
*空行*
## 一个损失的计算过程

**Comment by wangbo-zhao** [comment]
sorry. This is my first time to write a PR, and I have made a lot of mistakes. 😊

**Review by BIGWangYuDong — APPROVED** [review]

**Review by AronLin — APPROVED** [review]

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**